### PR TITLE
PakReq: Wesnoth

### PR DESCRIPTION
**Pull request by StephDC**
Battle of Wesnoth, a fantasy turn-based strategic game.
First packaging. May need to revise.

### CODE REVIEW

File: extra-games/wesnoth/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,7 @@
+PKGNAME=wesnoth
+PKGSEC=games
+PKGDEP="sdl sdl-image sdl-mixer sdl-ttf sdl-net"
+PKGDES="Battle of Wesnoth, a fantasy turn-based strategic game"
+PKGRECOM="fribidi"
+BUILDDEP="cmake boost"
+AB_FLAGS_FTF=0
```
Comment (MingcongBai) [inline extra-games/wesnoth/autobuild/defines]: Why?

File: extra-games/wesnoth/spec [hunk 2]
```diff
@@ -0,0 +1,3 @@
+VER=1.12.6
+SRCTBL="http://sourceforge.net/projects/wesnoth/files/wesnoth-${VER:0:4}/wesnoth-$VER/wesnoth-$VER.tar.bz2"
+wget $SRCTBL
```
Comment (MingcongBai) [inline extra-games/wesnoth/spec]: This line is unnecessary.

Comment (StephDC) [inline extra-games/wesnoth/spec]: oops that was for testing purpose.

File: extra-games/wesnoth/autobuild/defines (hunk 1 above)
Comment (StephDC) [inline extra-games/wesnoth/autobuild/defines]: Which one? FTF?

**Comment by StephDC** [comment]
Updated should be fine now...

**Pull request merged**